test/test_coverart_image.py

```diff
@@ -1,12 +1,12 @@
 #!/usr/bin/env python
 # coding: utf-8
 import os.path
-import unittest
-
 from test.picardtestcase import (
     PicardTestCase,
     create_fake_png,
 )
+import unittest
```

Comment:
I don't think this should be moved after `from test.picardtestcase import`, i think we should add test.* as a section, but something like (untested) :

```
known_test=test.*
sections=FUTURE,STDLIB,THIRDPARTY,QT,TEST,PICARD,LOCALFOLDER,RESOURCES,PICARD_UI
```

Reply:
Done in dd4da2aa474892ca858163cb8ebc57a14fdc13ce, I like this better. So far we had no clear structure used for these files.